flood_adapt/object_model/interface/site.py

```diff
@@ -44,7 +54,7 @@ class SfincsModel(BaseModel):
 
     csname: str
     cstype: Cstype
-    version: Optional[str] = ""
+    version: str = ""
```

Comment:
why is this non Optional anymore?

Reply:
So the `Optional` is a bit of a misnomer here. Because we give it a default value, it is actually still optional. With the optional attribute it can only ever be none if the user actually specified `Cstype(version=None)` which would be very strange. So functionally it is always a string, and removing the optional attribute makes it easier on the type checker. So practically we don't change anything. Does that make sense?